src/wp-includes/class-wp-editor.php

```diff
@@ -969,8 +969,8 @@ public static function print_default_editor_scripts() {
 		?>
 		<script type="text/javascript">
 		window.wp = window.wp || {};
-		window.wp.editor = window.wp.editor || {};
-		window.wp.editor.getDefaultSettings = function() {
+		window.wp.editor = window.wp.oldEditor = window.wp.oldEditor || {};
```

Comment:
This has the potential to overwrite the new `wp.editor`, right? That normally wouldn't happen as this js is outputted inline, but plugins may also inject that HTML in already loaded page.

Also think the same is needed before initializing the "standard" old editor as used on the old Edit Post screen.

Reply:
> This has the potential to overwrite the new wp.editor, right?

@azaozz correct. Note this PR isn't at fault – this is just how it works now. Here we only add `window.wp.oldEditor =`. I would love to get rid of the `window.wp.editor =` part as a next step, but  we need to deprecate it first.

Reply:
> Also think the same is needed before initializing the "standard" old editor as used on the old Edit Post screen.

I don't quite follow, what do you mean?

Reply:
Uh, sorry, was thinking ahead/about the next step. Switching of all old core js to work with `oldEditor` looks good.